readme.md

```diff
@@ -1,2 +1,393 @@
+    on how to handle problems.
+  </dd>
+</dl>
+
+[Slides](http://fr.slideshare.net/xdecock/php-performance-sflive-2010)  
+~~Video~~
+
+By [Xavier De Cock](https://connect.sensiolabs.com/profile/xdecock)  
+![github](icon/github.png) [@xdecock](https://github.com/xdecock)  
+![twitter](icon/twitter.png) [@xdecock](https://twitter.com/xdecock)
+
+---
+
+## Okapi meet symfony, symfony meet Okapi
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    The Okapi framework was initially developed by the Liip AG together
+    for use at local.ch as thin layer mainly handling XSLT transformation
+    and request mapping between the Java backend and the PHP based
+    frontend. It has evolved since then but has stayed true its heritage
+    of being the thinnest possible layer that combines various open source
+    libraries. In this spirit the sfYaml component was integrated in
+    version 1.1 of the framework. Okapi 2 has integrated various
+    additional symfony components to further reduce the amount of code
+    that has to be maintained in house, while at the same time giving
+    Okapi a very modern architecture having adopted the event dispatcher,
+    request handler, service container components. This sessions explains
+    why we choose to integrate these components and how we went about
+    implementing them and more importantly what lessons we learned along
+    the way.
+  </dd>
+</dl>
+
+~~Slides~~  
+~~Video~~
+
+By [Lukas Kahwe Smith](https://connect.sensiolabs.com/profile/lsmith)  
+![github](icon/github.png) [@lsmith77](https://github.com/lsmith77)  
+![twitter](icon/twitter.png) [@lsmith](https://twitter.com/lsmith)
+
+---
+
+## Introduction to Git
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    This talk will cover what Git is and how it differs from Subversion or
+    Perforce. We will go over why it’s so powerful and how it turns source
+    control from a necessary annoyance into a powerful development tool.
+    We will cover the technical underpinnings of the system, common tips
+    and best practices for getting the most out of Git and ways to use Git
+    that you may not have thought of before – as a pure
+    content-addressable filesystem. Attendees will leave with a better
+    understanding of Git and hopefully a new way of thinking about how to
+    work.
+  </dd>
+</dl>
+
+~~Slides~~  
+~~Video~~
+
+By Scott Chacon  
+![github](icon/github.png) [@schacon](https://github.com/schacon)  
+![twitter](icon/twitter.png) [@chacon](https://twitter.com/chacon)
+
+---
+
+## Implementing a symfony-based CMS in a publishing company
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    In this session we'll talk about implementing a Symfony-based CMS
+    system in an editorial company. Will be based on our experience
+    building web CMS solutions for traditional publishing companies
+    and will cover all steps on this kind of projects.
+  </dd>
+</dl>
+
+~~Slides~~  
```

Comment:
http://slideshare.net/esmiz/implementing-a-symfony-based-cms-in-a-publishing-company